scripts/migrations/002-remove-webview-flags.js

```diff
@@ -0,0 +1,98 @@
+#!/usr/bin/env node
+/* Any copyright is dedicated to the Public Domain.
+ * http://creativecommons.org/publicdomain/zero/1.0/ */
+
+'use strict';
+const fs = require('fs');
+const path = require('path');
+const { platform } = require('os');
+
+/** Determines if the OS is Windows */
+const IS_WINDOWS = platform() === 'win32';
+
+const removeWebViewFlags = (key, value) => {
+  if (key === "__compat") {
+    if (value.support.webview_android !== undefined) {
+      if (Array.isArray(value.support.webview_android)) {
+        var result = [];
+        for (var i = 0; i < value.support.webview_android.length; i++) {
```

Comment:
Is there a reason you use `var` in this function (as compared to `let` elsewhere)?

Reply:
Old habits die hard -- I'm more accustomed to using `var` myself from when I first started JavaScript, so I have no other reason.  Switching this over now!